tests/shared/process-event.ts

```diff
@@ -222,7 +222,7 @@ export const createProcessEventTests = (
                     $browser: 'Chrome',
                     $current_url: 'https://test.com',
                     $os: 'Mac OS X',
-                    $browser_version: 80,
+                    $browser_version: false,
```

Comment:
Why this change?

Reply:
This tests the case you highlighted where $set_once would be set again if the property was false. It also tests the other issue you highlighted with partial saves.